### PR TITLE
Fixed urllib for python3

### PR DESCRIPTION
**Pull request by hrkrx**
Removed unnecessary imports
Fixed method calls to urlencode

### CODE REVIEW

**Review by PumpkinCheshire — COMMENTED** [review]
From python 3.5, you still need to import urllib.request and urllib.parse or those modules will not be found.

**Review by samwhipmedia — APPROVED** [review]

**Comment by samwhipmedia** [comment]
Sorry approved by accident, @hrkrx can you update to include import of urllib.request and urllib.parse?

**Review by samwhipmedia — CHANGES_REQUESTED** [review]
per comment, additional import needed.

File: tvdb_v4_official.py [hunk 1]
```diff
@@ -337,12 +333,12 @@ def get_updates(self, since: int) -> list:
...
     def get_all_tag_options(self, page=0) -> list:
         """Returns a list of tag options"""
-        url = self.url.tag_options_url(page)
+        url = self.url.tag_option_url(page)
         return self.request.make_request(url)
 
     def get_tag_option(self, id: int) -> dict:
         """Returns a tag option dictionary"""
-        url = self.url.tag_option_url()
+        url = self.url.tag_options_url()
```
Comment (WayneD) [inline tvdb_v4_official.py:341]: This shouldn't change the url function name, but instead it should pass the `id` arg:
```suggestion
        url = self.url.tag_option_url(id)
```

File: tvdb_v4_official.py [hunk 2]
```diff
@@ -337,12 +333,12 @@ def get_updates(self, since: int) -> list:
 
     def get_all_tag_options(self, page=0) -> list:
         """Returns a list of tag options"""
-        url = self.url.tag_options_url(page)
+        url = self.url.tag_option_url(page)
```
Comment (WayneD) [inline tvdb_v4_official.py:336]: This shouldn't change the url function name. Instead, the tag_options_url() function needs to accept a page arg. If you want to do something with the page arg, the url assignment in that function could also be changed, as in the second line of this suggested function:

```
   def tag_options_url(self, page=0):
        url = "{}/tags/options?page={}".format(self.base_url, page)
```

**Review by WayneD — CHANGES_REQUESTED** [review]
I believe the  tags options changes aren't right.

File: tvdb_v4_official.py [hunk 3]
```diff
@@ -1,8 +1,5 @@
 import json
 import urllib
-import urllib.parse
-import urllib.request
```
Comment (WayneD) [inline tvdb_v4_official.py:4]: When I tested with python 3.8.10 it still requires an "import urllib.request" for the script to work.

**Pull request closed without merging**